.github/workflows/build-docs.yml

```
@@ -0,0 +1,79 @@
+name: "Pull Request Docs Check"
+on: 
+- pull_request
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out repository code
+      uses: actions/checkout@v3
+    - name: Install prerequisites
+      run: | 
+        sudo apt-get --allow-releaseinfo-change update -y && sudo apt-get install -y tox
+    - name: Building GOV and common
+      run: |
+        cd doc
+        echo "---------------------------------------------------------------------------------------"
+        echo "---------------------------------- Building GOV ---------------------------------------"
+        echo "---------------------------------------------------------------------------------------"
+        tox -e docs
+        echo "Building GOV was succesfull! "

```

Not adding value, just nitpicking ;-)

```suggestion
        echo "Building GOV was successful! "

```